gen/calc.py

```diff
@@ -371,6 +371,7 @@ def validate_os_type(os_type):
         'ui_banner_footer_content': 'null',
         'ui_banner_image_path': 'null',
         'ui_banner_dismissible': 'null',
+        'dcos_overlay_config_attempts': '4',
```

Comment:
Well, we don't have a minimum here. So if the user sets it to 0, there is nothing wrong, the first configuration error would cause a failure, which should be acceptable.

I will however add a validation to check it is a valid integer. Does that make sense?